tasks/release.py

```diff
@@ -1020,7 +1020,7 @@ def finish(ctx, major_versions="6,7"):
 
     for major_version in list_major_versions:
         new_version = next_final_version(ctx, major_version)
-        update_release_json(github_token, new_version)
+        update_release_json(ctx, github_token, new_version)
```

Comment:
This is more of a functional error than a security alert
https://github.com/stephengroat/datadog-agent/security/code-scanning/277